docs/content/stable/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -54,6 +54,20 @@ Specify one of the following actions.
 
 Add the specified column with the specified data type and constraint.
 
+##### Table rewrites
+
+ADD COLUMN … DEFAULT statements require a [table rewrite](#alter-table-operations-that-involve-a-table-rewrite) when the default value uses *volatile* functions. [Volatile functions](https://www.postgresql.org/docs/current/xfunc-volatility.html#XFUNC-VOLATILITY) can return different results for different rows, so a table rewrite is required to fill in values for existing rows. For non-volatile functions, no table rewrite is required.
```

Comment:
nit: i think volatile expressions is more accurate (similarly non-volatile expressions)

Reply:
when the default value is a volatile expression?

Reply:
sounds good, will do.